hazelcast/src/main/java/com/hazelcast/jet/impl/MasterJobContext.java

```diff
@@ -313,7 +313,6 @@ private DAG resolveDag(Supplier<Long> executionIdSupplier) {
             if (mc.jobExecutionRecord().isSuspended()) {
                 mc.jobExecutionRecord().clearSuspended();
                 mc.writeJobExecutionRecord(false);
-                mc.setJobStatus(NOT_RUNNING);
```

Comment:
I think that this clears some inconsistent data after coordinator crash. The job may not be immediately started (eg. due to lack of quorum). Is this line really redundant?

Reply:
It is redundant because of the above condition: `if (mc.jobStatus() != NOT_RUNNING) return null;`